packages/contracts-bedrock/scripts/upgrades/PostSherlockL2.s.sol

```diff
@@ -126,6 +126,21 @@ contract PostSherlockL2 is SafeBuilder {
         require(PROXY_ADMIN.getProxyImplementation(prox.OptimismMintableERC721Factory).codehash == impl.OptimismMintableERC721Factory.codehash);
     }
 
+    /**
+     * @notice The address of the proxy admin.
+     */
+    function getProxyAdmin() public pure override returns (address) {
+        return 0x4200000000000000000000000000000000000018;
```

Comment:
Hmm the proxy admin and safe were originally the 2 args to the `run` function. Ideally we do not couple these scripts to our network more and we can move towards having scripts that are agnostic to the network. To do this, we should be moving towards less hardcoding and more reading from a config file

Reply:
LMK if I should introduce a config file in this pr.